dashboard/src/actions/namespace.ts

```diff
@@ -55,7 +55,11 @@ export function fetchNamespaces(
   return async dispatch => {
     try {
       const namespaceList = await Namespace.list(cluster);
-      const namespaceStrings = namespaceList.namespaces.map((n: IResource) => n.metadata.name);
+
+      const namespaceStrings =
+        namespaceList.namespaces === undefined
+          ? []
+          : namespaceList.namespaces.map((n: IResource) => n.metadata.name);
```

Comment:
Did we stop using lodash.get? This would be simpler as something like `get(namespaceList, "namespaces", []).map(...)` ?

Reply:
Ooops, yep, we still use it... but Lodash is not the first thing coming to my mind. Thanks for the suggestion!